mmcv/ops/nms.py

```diff
@@ -289,51 +289,60 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
     Returns:
         tuple: kept dets and indice.
     """
-    nms_cfg_ = nms_cfg.copy()
-    class_agnostic = nms_cfg_.pop('class_agnostic', class_agnostic)
-    if class_agnostic:
-        boxes_for_nms = boxes
+    # skip nms when nms_cfg is None
+    if nms_cfg is None:
+        scores, inds = scores.sort(descending=True)
+        boxes = boxes[inds]
+        return torch.cat([boxes, scores[:, None]], -1), inds
     else:
```

Comment:
Can we omit such an else and keep the rest of the code unchanged?

Reply:
ok， I am just used to adding an `else` after `if`